custom/lib_portal/models/res_partner.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = 'res.partner'
+    
+    def _isMember(self):
+        """
+        this Function uses to get the ratification amount in words.
+        """
```

Comment:
* this description seems to be unfitting for this method

Reply:
This is mistake from my end when I am copying it.I have changed it now.